File: 8_Kyu_Kata/how_good_are_you_really/Python/how_good_are_you_really.py
# 8 kyu - How good are you really?  [ ID: 5601409514fc93442500010b  (how-good-are-you-really) ]
# URL: https://www.codewars.com/kata/5601409514fc93442500010b
# Category: NONE  |  Tags: FUNDAMENTALS
# ******************************************************************************
def better_than_average(class_points, your_points):
      return sum(class_points, your_points) / (len(class_points) + 1) < your_points

File: 8_Kyu_Kata/how_good_are_you_really/JavaScript/howGoodAreYouReally.js
// 8 kyu - How good are you really?  [ ID: 5601409514fc93442500010b  (how-good-are-you-really) ]
// URL: https://www.codewars.com/kata/5601409514fc93442500010b
// Category: undefined  |  Tags: FUNDAMENTALS
// *****************************************************************************
const betterThanAverage = (classPoints, yourPoints) =>
  yourPoints > (classPoints.reduce((a, b) => a + b) + yourPoints) / (classPoints.length + 1)
